packages/gp2-components/src/molecules/EventOwner.tsx

```diff
@@ -0,0 +1,56 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { Link, neutral900, pixels } from '@asap-hub/react-components';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import { projectIcon, workingGroupIcon } from '../icons';
+import IconWithLabel from './IconWithLabel';
+
+const { rem } = pixels;
+
+const { projects, workingGroups } = gp2Routing;
+const listItemStyles = css({
+  padding: `${rem(7.5)} 0`,
+  color: neutral900.rgb,
+  whiteSpace: 'break-spaces',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  fontSize: rem(17),
+  display: 'flex',
+});
+
+type EventOwnerProps = Pick<gp2Model.EventResponse, 'project' | 'workingGroup'>;
+const getProperties = ({ project, workingGroup }: EventOwnerProps) => {
+  if (workingGroup) {
+    return {
+      ...workingGroup,
+      href: workingGroups({}).workingGroup({
+        workingGroupId: workingGroup.id,
+      }).$,
+      icon: workingGroupIcon,
+    };
+  }
+  return (
+    project && {
+      ...project,
+      href: projects({}).project({ projectId: project.id }).$,
+      icon: projectIcon,
+    }
+  );
+};
+
+const EventOwner: React.FC<EventOwnerProps> = ({ project, workingGroup }) => {
+  const { title, href, icon } = getProperties({ project, workingGroup }) || {
```

Comment:
getProperties can check and return the default props, we don't have to do it here

Reply:
done